test/samples/test.samples.jobs.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018, Google, LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as assert from 'assert';
+import * as nock from 'nock';
+import {Utils} from '../utils';
+
+nock.disableNetConnect();
+
+// tslint:disable: no-any
+const samples: any = {
+  jobs: require('../../../samples/jobs/jobs.js'),
+};
+
+for (const p in samples) {
+  if (samples[p]) {
+    samples.jobs.client.credentials = {access_token: 'not-a-token'};
+  }
+}
+
+describe('Talent API Samples', () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  it('should create a company', async () => {
+    const scope = nock(Utils.baseUrl)
+                      .post(`/v3/jobs/project/project-id/companies`)
```

Comment:
Ah! I think this is the problem.  For this url, can you use `${process.env.GCLOUD_PROJECT}`?

Reply:
Good idea! I tried it several different ways though and I'm still getting:
```
FetchError: request to https://jobs.googleapis.com/v3/project/project-id/companies failed, 
reason: Nock: Disallowed net connect for "jobs.googleapis.com:443/v3/project/project-id/companies"
```
🎊

I accepted the invitation (thanks!), I'll be  pushing to a branch soon

Reply:
huh weird.  Share your updated code?

Reply:
Sure! Here's version 1:
```
  it('should create a company', async () => {
    const scope = nock(Utils.baseUrl)
                      .post(`/v3/jobs/project/${process.env.GCLOUD_PROJECT}/companies`)
```
version 2:
```
  it('should create a company', async () => {
    const scope = nock(Utils.baseUrl)
                      .post(`${process.env.GCLOUD_PROJECT}`)
```

Reply:
Huh, v1 looks right to me

Reply:
Some logging revealed that `GCLOUD_PROJECT` is `undefined` until we go into runSample's scope where we call on the auth service to generate a project Id - copying the same code (` const projectId = await google.auth.getProjectId();`) into the test's scope gives us `undefined` too so.. any chance we can erase this test and push the sample up anyway? A bit anarchistic but I noticed there are a few other APIs without them

Reply:
Keep the test, just put a `.skip` on it :)

Reply:
Cool! Done!